index.js

```diff
@@ -1,8 +1,9 @@
 import fs from "fs";
 import path from "path";
 import partitionAll from 'partition-all';
-import {BrowserFactory, SAUCELABS_CONNECTION} from "./src/BrowserFactory";
-import {CapabilityFactory} from "./src/CapabilityFactory";
+import { SLBrowserFactory, SAUCELABS_CONNECTION  } from "./src/SLBrowserFactory";
```

Comment:
Because their interfaces are the same, you can rename the exports in both SLBrowserFactory and TBBrowserFactory to `BrowserFactory` and `CONNECTION` and also move the options and import them then you can revert the changes with the variable namespacing here:

```js
import { BrowserFactory, CONNECTION, factoryOptions } from "./src/SLBrowserFactory";
// or
import { BrowserFactory, CONNECTION, factoryOptions } from "./src/TBBrowserFactory";
```
This will make the code more generic.